mito-ai/mito_ai/models.py

```diff
@@ -237,3 +235,61 @@ class FetchHistoryReply:
     # Message type.
     type: Literal["reply"] = "reply"
 
+@dataclass(frozen=True)
+class ChatThreadItem:
```

Comment:
Maybe we should call this `ChatThreadMetadata`? `Item` makes it sound to me like its the entire ChatThread